jenkins/onmetal_simple_upgrade.pipeline

```diff
@@ -152,6 +152,10 @@ node('onmetal-provisioner') {
     osa.run_persistent_resources_tests('verify', 'after_upgrade')
     osa.run_persistent_resources_tests('cleanup')
     
+    // Pase persistent resource results
+    osa.setup_parse_persistent_resources()
+    osa.parse_persistent_resources_tests()
```

Comment:
Ok, thanks for the explanation.
Two more things:
1) I think it would make more sense to add this into the reporting stage instead of being in the post upgrade validation stage.
2) You need t rebase, there are conflicts with your patch now.